examples/community/k_stable_diffusion.py

```diff
@@ -0,0 +1,354 @@
+import inspect
+from typing import Callable, List, Optional, Union
+
+import torch
+
+from transformers import CLIPFeatureExtractor, CLIPTextModel, CLIPTokenizer
+
+from diffusers.configuration_utils import FrozenDict
+from diffusers import AutoencoderKL, DiffusionPipeline, DDIMScheduler, LMSDiscreteScheduler, PNDMScheduler, UNet2DConditionModel
+from diffusers.pipelines.stable_diffusion import StableDiffusionPipelineOutput
+from diffusers.utils import deprecate, logging
+from diffusers.pipelines.stable_diffusion.safety_checker import StableDiffusionSafetyChecker
+
+import k_diffusion as K
+import torch.nn as nn
+
+class KWrapper(nn.Module):
```

Comment:
Maybe we could also add this here: https://github.com/crowsonkb/k-diffusion/blob/f4e99857772fc3a126ba886aadf795a332774878/k_diffusion/external.py#L131 -> looks nice!